packages/next-frontend/src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from "react";
```

Comment:
If you want I think using `react-use` is perfectly acceptable, since it should be fully tree-shakeable

Reply:
Yes I ended up implementing the debounce to avoid bringing the whole `react-user` library, but if it's tree-shakeable there should be no issues.

Reply:
Fixed in a23af6205ab6c8645751564c0f9ba1778575f67d.